src/current/molt/molt-fetch.md

```diff
@@ -82,6 +82,8 @@ Complete the following items before using MOLT Fetch:
 
 - When using [`IMPORT INTO`](#data-movement) to load tables into CockroachDB, if the fetch task terminates before the import job completes, the hanging import job on the target database will keep the table offline. To make this table accessible again, [manually resume or cancel the job]({% link {{site.current_cloud_version}}/import-into.md %}#view-and-control-import-jobs). Then resume `molt fetch` using [continuation](#fetch-continuation), or restart the task from the beginning.
 
+- Ensure that the machine running MOLT Fetch is large enough to handle the amount of data being migrated. Fetch performance can sometimes be limited by available resources, but should always be making progress. To identify possible resource constraints, set `--logging debug` and observe the [metrics endpoint](#metrics).
```

Comment:
> To identify possible resource constraints, set `--logging debug`
When reading this, i'm not quite sure what to interpret from this. Is it to say that with debug logging we should see incremental progress?

Reply:
We need to communicate:
1. the machine needs to be big enough
2. perf can slow to a crawl if there are resource constraints
3. the best way to know if you're having resource constraints is by creating a counter on the rows processed metric that measures the rate of change of rows processed

Reply:
Re 3. We already have that metric. You just have to look at it.

Reply:
We have a sample json dashboard here, just not sure how we can include this in docs. https://github.com/cockroachdb/molt/blob/main/docker/grafana/dashboard.json

Reply:
Re: 3, are we talking about `molt_fetch_rows_exported` or `molt_fetch_rows_imported`?

Can we make the JSON dashboard downloadable on molt.cockroachdb.com like with the release binaries? If it's something with broader functionality I'd like to detail exactly what's in it. I suggest saving that for a separate issue.

Reply:
It would be for molt_fetch_rows_exported.

I think we can add it. @ryanluu12345 We can just uploade the dashboard.json into our manifest bucket ourself right?

Reply:
>Re 3. We already have that metric. You just have to look at it.
my point is that we need to be explicit in our docs

Reply:
Updated.

Reply:
I've confirmed that `molt_fetch_rows_exported` updates as we CSV pipe the data we read from the source database. So it is correct to look at that metric. 

Regarding dashboard.json, yeah we can upload it to the bucket. But we don't have permission to manually do that, will need to ask dev-inf. Also open questions:
1. Do we want to just keep one version of this at all times?
2. Do we want to automate this?
3. If we automate this, do we want to publish a new one each time for the release? Or is the intention here to pull the dashboard at time of release and throw in the zip?

Reply:
Discussed offline here: we are going to just upload the latest dashboard.json whenever we release and put it in the `molt/cli/` directory. I'll give the full path in my PR and Ryan can add docs here.

Reply:
@taroface @rohan-joshi @Jeremyyang920 , please review here: https://github.com/cockroachdb/molt/pull/424/files

I've gone ahead and added this into our CI release process so that upon release, we upload the latest version of the dashboard to our GCS bucket. This will then be accessible to users inside that same bucket path but at a different file name identifier. Please let me know your thoughts here.

Reply:
Confirmed that based on the linked PR, it should live here: https://molt.cockroachdb.com/molt/cli/grafana_dashboard.json

Reply:
Added this to the Fetch doc.